airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/report_streams.py

```diff
@@ -118,6 +116,7 @@ def __init__(self, config: Mapping[str, Any], profiles: List[Profile], authentic
         self._session = requests.Session()
         self._model = self._generate_model()
         self._start_date: Optional[Date] = config.get("start_date")
+        self._look_back_window: int = config.get("look_back_window")
```

Comment:
because `look_back_window` not optional can you please 
change `config['look_back_window']`
?